src/hooks/useCopyToClipboard.js

```diff
@@ -16,7 +16,7 @@ export const copyToClipboard = (text) => {
 };
 
 const useCopyToClipboard = ({ text, successDuration }) => {
-  const [isCopied, setIsCopied] = useState(false);
+  const [isCopied, setIsCopied] = useState(() => false);
```

Comment:
`useState(() => false)`

Why such things still here?

Reply:
https://reactjs.org/docs/hooks-reference.html#lazy-initial-state
Because this is not a mistake?